src/pages/debug-app/debug-app.tsx

```diff
@@ -163,15 +169,17 @@ export const DebugApp = () => {
   const [examples, setExamples] = useState<LayerExample[]>(EXAMPLES);
   const [activeLayers, setActiveLayers] = useState<LayerExample[]>([]);
   const [viewState, setViewState] = useState<ViewStateSet>(INITIAL_VIEW_STATE);
-  const fetchSublayersCounter = useRef<number>(0);
+  const fetchSublayersCounter = useRef<number>(
```

Comment:
I believe we can serve this logic with "counter" inside the redux

Reply:
Done